### PR TITLE
Create report form

### PR DESCRIPTION
**Pull request by apiatski**

### CODE REVIEW

**Comment by sandlerben** [comment]
Do you want another review of this PR?

**Comment by apiatski** [comment]
Yes, please

**Comment by sandlerben** [comment]
Can you make another PR with a base of basic_map? Like I told you on your earlier PR, it's hard to review this because it includes a bunch of changes which you don't introduce.

**Comment by sandlerben** [comment]
Kinda nit: Some of your commit messages could be improved. Commit messages like "pep8 stuff 2" and "Changes based on Ben's method" don't tell me much. It's also not a good practice to have a bunch of separate commits with the same messages. I would use `git rebase -i` to clean up the git history a bit.

**Comment by sandlerben** [comment]
Also are you using viewport biasing with geocoding?

File: app/main/forms.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+from wtforms import validators
+from ..models import Agency
+from wtforms.ext.sqlalchemy.fields import QuerySelectField
+from wtforms.validators import DataRequired
+from .. import db
+import datetime as datetime
+
+
+class NewIncidentForm(Form):
+    license_plate_number = StringField('License Plate Number', [
```
Comment (sandlerben) [inline app/main/forms.py:13]: I'm pretty sure license_plate shouldn't be required

Comment (kongsally) [inline app/main/forms.py:13]: Yeah, I think it was Vehicle ID that was required and License Plate Number was optional. 

Fields: 
- Agency 
- Vehicle ID 
- License Plate Number
- Description (ex. Driver was sleeping)
- Idling Duration (minutes) 
- Location (people will put down a pin)
- Photo

**Comment by sandlerben** [comment]
Note: I'm changing the main Incident_Report_Form to have both a picture_file field and picture_url field. I'm doing this because I'm using the validators in this form in csv parsing and twilio stuff. Someone (probably me) needs to reincorporate master and explicitly write which fields should render in templates.

**Comment by sandlerben** [comment]
Also, I'm not sure that we need to show the `date`/`time` fields. Maybe it should just set to right now?

**Comment by sandlerben** [comment]
@apiatski I actually don't want to review until we merge master in. Once @kongsally does that, I can review the code.

**Comment by kongsally** [comment]
@apiatski Are we supposed to be reviewing for this branch's PR? Or the working_forms_updated?

**Comment by sandlerben** [comment]
See #84

**Pull request closed without merging**